policyengine_us/variables/gov/states/ri/tax/income/exemption/ri_exemptions.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class ri_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island exemptions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022%20Tax%20Rate%20and%20Worksheets.pdf"
+    defined_for = StateCode.RI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ri.tax.income.exemption
+
+        exemptions = tax_unit("exemptions", period)
```

Comment:
`exemptions` is now an amount (may have been the count when you started this pr)
```suggestion
        exemptions_count = tax_unit("exemptions_count", period)
```